arwen/elrondapi/bigFloatOps.go

```diff
@@ -543,7 +542,7 @@ func pow(context unsafe.Pointer, base *big.Float, exp int32) (*big.Float, error)
 }
 
 //export v1_4_bigFloatFloor
-func v1_4_bigFloatFloor(context unsafe.Pointer, opHandle, destBigIntHandle int32) {
+func v1_4_bigFloatFloor(context unsafe.Pointer, destBigIntHandle, opHandle int32) {
```

Comment:
Using consecutive arguments of the same type is always tricky:
`func v1_4_bigFloatFloor(context unsafe.Pointer, destBigIntHandle, opHandle int32)`

This is prone to the following programming error:
v1_4_bigFloatFloor(context, opHandle, destBigIntHandle)` 
which the compiler does not catch (in many cases not even the reviewers)